src/datasource.ts

```diff
@@ -535,14 +535,8 @@ export class OpenSearchDatasource extends DataSourceWithBackend<OpenSearchQuery,
     if (
       request.targets.every(
         (target) =>
-          target.metrics?.every(
-            (metric) =>
-              metric.type === 'raw_data' ||
```

Comment:
I think we are losing these types `raw_data` and `raw_document` or am I not reading this right? I wonder if it would be easier at this point to say what doesn't go through the backend flow rather than what does?

Reply:
The `raw_data` and `raw_document` are Lucene queries that we never restricted to execute on the backend in Explore only.

I've updated with your suggestion to check what shouldn't go through the backend flow, which should be
1. PPL query
2. formatted as a time series
3. run from the dashboard